packages/core/stories/layout/flex-layout.stories.tsx

```diff
@@ -50,7 +57,12 @@ export default {
       control: "boolean",
     },
   },
-  excludeStories: ["ContactDetailsExample", "FlexLayoutNestedExample"],
+  excludeStories: [
```

Comment:
Maybe not for this PR but ContactDetailsExample isn't used anywhere in this file so not really sure if it belongs here?